packages/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -1,8 +1,10 @@
+import { ComponentProps, ObjectShorthandProps, ShorthandProps } from '@fluentui/react-utils';
 import * as React from 'react';
+
 import { BadgeProps } from '../Badge/index';
-import { ShorthandValue } from '../utils/commonTypes';
+import { ImageProps } from '../Image/index';
 
-export interface AvatarProps extends React.HTMLAttributes<HTMLElement> {
+export interface AvatarProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
```

Comment:
For example, there's a test in Avatar.test.tsx that should cause a build break in this PR because it uses the `tokens` prop, but wasn't caught as a result of this.

Reply:
Good catch 👍 I think that we should use this type everywhere to be consistent, an issue with `any` & `GenericDictionary` we have fixed in #16902.